common/src/main/scala/explore/GraphQLSchemas.scala

```diff
@@ -64,4 +66,21 @@ object GraphQLSchemas {
         explore.model.encoders.siderealTargetEncoder
     }
   }
+
+  @GraphQLSchema(debug = false)
+  object UserPreferencesDB {
+    object Scalars {
+      type UserId        = User.Id
+      type ResizableArea = String
+    }
+
+    object Types {
+      type WidthInsertInput = ExploreResizableWidthInsertInput
+      type WidthInsertInputL = Array[ExploreResizableWidthInsertInput]
+      implicit final val jsonEncoderWidthInsertInput: io.circe.Encoder[WidthInsertInput] =
+        ???
+      implicit final val jsonEncoderWidthInsertInputL: io.circe.Encoder[Array[WidthInsertInput]] =
+        ???
```

Comment:
`clue` macro should have defined `Encoder[ExploreResizableWidthInsertInput]` (in the `Implicits` object), so this may not be necessary.